src/main/java/me/partlysanestudios/partlysaneskies/config/keybinds/Keybinds.java

```diff
@@ -63,7 +63,7 @@ private static KeyBinding registerKey(String name, String category, int keycode)
 
     @SubscribeEvent
     public void keybindWhileInGui(KeyboardInputEvent.Post event) {
-        if (PartlySaneSkies.config.debugKeybind.isActive()) {
+        if (PartlySaneSkies.Companion.getConfig().debugKeybind.isActive()) {
```

Comment:
Honestly we might need to store "config" and "minecraft" outside of the main in an object now

Reply:
Our quantity of java code is only going to decrease

Reply:
Yeah realized it in review no3